package.json

```diff
@@ -41,6 +41,7 @@
     "babel-plugin-webpack-alias-7": "^0.1.1",
     "eslint": "^5.12.1",
     "eslint-loader": "^2.1.1",
+    "mock-require": "^3.0.3",
```

Comment:
I don't think we need this dependency anymore, right?

Reply:
Yeah